features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionEndedMetric.kt

```diff
@@ -50,8 +50,9 @@ internal class SessionEndedMetric(
         return true
     }
 
-    fun onErrorTracked(sdkErrorKind: String) {
-        errorKindFrequencies[sdkErrorKind] = (errorKindFrequencies[sdkErrorKind] ?: 0) + 1
+    fun onErrorTracked(sdkErrorKind: String?) {
+        val errorKindKey = sdkErrorKind ?: SDK_ERROR_DEFAULT_KIND
+        errorKindFrequencies[errorKindKey] = (errorKindFrequencies[sdkErrorKind] ?: 0) + 1
```

Comment:
tiny mistake here, I believe

```suggestion
        errorKindFrequencies[errorKindKey] = (errorKindFrequencies[errorKindKey] ?: 0) + 1
```

Reply:
fixed